peachjam/forms.py

```diff
@@ -380,3 +380,12 @@ class PeachjamSignupForm(SignupForm):
 
 class PeachjamLoginForm(LoginForm):
     captcha = ReCaptchaField(widget=ReCaptchaV2Invisible)
+
+
+class UserEditorForm(forms.ModelForm):
+    first_name = forms.CharField(label=_("First name"))
```

Comment:
you should be able to drop both of these and just use what the form determines from the object's properties.

Also the class name would usually be `UserForm`